### ISSUE TITLE
Some widgets not updating appropriately on style change

### ISSUE THREAD

**Issue by jamesobutler** [Slicer/Slicer#5135]
There are a few widgets that aren't updating when changing Slicer Style and then continuing to use the session. If I find anymore examples and this issue is still open, I will add them here.

- The qMRMLMarkupsDisplayNodeWidget in the Markups module:
![image](https://user-images.githubusercontent.com/15837524/91217146-0c828100-e6e5-11ea-916c-7e0900f2d34a.png)

- The qMRMLSegmentationRepresentationsListView in the Segmentations module:
![image](https://user-images.githubusercontent.com/15837524/91217264-38056b80-e6e5-11ea-856f-1d63927de319.png)

**Comment by pieper** [comment]
Similar issue: what I believe is called the "WelcomeText" for the python console shows up very dark in dark mode (top) but you can see that it's there if you highlight it (bottom)

![image](https://user-images.githubusercontent.com/126077/91218832-a64b2d80-e6e7-11ea-8400-3bb0c7bab814.png)

![image](https://user-images.githubusercontent.com/126077/91218841-ad723b80-e6e7-11ea-8470-2a83330a92e8.png)

**Comment by lassoan** [comment]
Currently, console text is not expected to update dynamically (only new text). If it is a problem then we should probably add a separate issue for it.

**Comment by pieper** [comment]
Yes, this is a different issue because this is a startup issue and not related to switching styles on the fly.

https://github.com/Slicer/Slicer/issues/5137

**Comment by jamesobutler** [comment]
The palette issue with the "Display" section of the markups module is due to issues with using the qMRMLCollapsibleButton object. Appears that usually the ctkCollapsibleButton is used instead. An obvious quick solution would be to replace with a ctkCollapsibleButton.

It appears that qMRMLCollapsibleButton is also used in the Texts module and the same palette issue can be observed there.
![image](https://user-images.githubusercontent.com/15837524/111225094-bceaa080-85b5-11eb-8c33-0acaf8d60c52.png)

This palette issue is caused by the code at:
https://github.com/Slicer/Slicer/blob/a3cf4cffa74bf7fcda005b3d737b6ac12ae2f0e7/Libs/MRML/Widgets/qMRMLCollapsibleButton.cxx#L49-L69

It appears qMRMLCollapsibleButton.cxx hasn't been modified in over 9 years. Based on what little the class is doing does it make sense to even keep qMRMLCollapsibleButton around? I'm not sure how MRML logic would be used in this widget. cc: @lassoan @pieper @lassoan.

**Comment by pieper** [comment]
I don't recall ever using this widget and wouldn't miss it.  I've always tried to avoid the custom palette manipulations so I'm not sure what behavior this code was trying to achieve.  @jcfr might have an idea?
